Modules/Filtering/BinaryMathematicalMorphology/include/itkBinaryErodeImageFilter.h

```diff
@@ -27,10 +27,14 @@ namespace itk
 {
 /**
  * \class BinaryErodeImageFilter
- * \brief Fast binary erosion
+ * \brief Fast binary erosion of a single intensity value in the image.
  *
  * BinaryErodeImageFilter is a binary erosion
- * morphologic operation. This implementation is based on the papers:
+ * morphologic operation on the foreground of an image. Only the value designated
+ * by the single "DilateValue" is considered as foreground, and all other values
+ * are considered background.
```

Comment:
What value are the eroded pixels replaced with?

Reply:
I would assume zero.

Reply:
m_BackgroundValue = NumericTraits<OutputPixelType>::NonpositiveMin(); 

and it can be overridden.